cms/envs/aws.py

```diff
@@ -210,6 +211,18 @@
 TENDER_DOMAIN = ENV_TOKENS.get('TENDER_DOMAIN', TENDER_DOMAIN)
 TENDER_SUBDOMAIN = ENV_TOKENS.get('TENDER_SUBDOMAIN', TENDER_SUBDOMAIN)
 
+# Modules having these categories would be excluded from progress calculations
```

Comment:
As noted in PR #673, please include more detail about why we are excluding these categories